librz/asm/arch/riscv/riscv.c

```diff
@@ -36,23 +36,23 @@
 #define ARRAY_SIZE(a) (sizeof(a) / sizeof(*a))
 
 // TODO : an conf to chose between abi or numeric
-static const char * const *riscv_gpr_names = riscv_gpr_names_abi;
-static const char * const *riscv_fpr_names = riscv_fpr_names_abi;
+static const char *const *riscv_gpr_names = riscv_gpr_names_abi;
+static const char *const *riscv_fpr_names = riscv_fpr_names_abi;
 static int init = 0;
 
-static void arg_p (char *buf, unsigned long val, const char* const* array, size_t size) {
+static void arg_p(char *buf, unsigned long val, const char *const *array, size_t size) {
```

Comment:
I wonder why it wasn't caught by the `clang-format`...

Reply:
I was thinking the same, the file was oddly formatted

Reply:
> I wonder why it wasn't caught by the `clang-format`...

because it is gpl.